ui/src/data/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -100,6 +102,7 @@ const DetailPage = ({
                     page="Data detail"
                   />
                 </div>
+                <LiteratureDate date={date} />
```

Comment:
this will render an empty span if no date here. Although it is very unlikely, we can do {date && ....}